homeassistant/components/zwave_js/manifest.json

```diff
@@ -30,5 +30,8 @@
     }
   ],
   "zeroconf": ["_zwave-js-server._tcp.local."],
-  "loggers": ["zwave_js_server"]
+  "loggers": ["zwave_js_server"],
+  "supported_brands": {
+    "leviton": "Leviton Decora"
```

Comment:
There is already an integration with that **exact** same name. It will be indistinguishable for the end-user.

Reply:
```suggestion
    "leviton": "Leviton"
```

Reply:
Ah right. Yea the integration docs page will be Leviton. So I think this is okay ?